README.md

```diff
@@ -39,55 +39,59 @@ Installation
 
 The following was tested on the Autometa [Docker](https://www.docker.com) image, which is based on the [Anaconda Docker image](https://hub.docker.com/r/continuumio/anaconda/). These instructions should work on [Debian](https://www.debian.org)-based linux distros such as [Ubuntu](https://www.ubuntu.com).
 
-First we install Prodigal, HMMER and DIAMOND.
+First install [miniconda](https://docs.conda.io/en/latest/miniconda.html# "miniconda).
 
-```
-sudo apt-get install prodigal hmmer build-essential zlib1g-dev
-mkdir diamond
-cd diamond
-wget http://github.com/bbuchfink/diamond/releases/download/v0.9.14/diamond-linux64.tar.gz
-tar xvf diamond-linux64.tar.gz
+```bash
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh
 ```
 
-At this point you should add the diamond directory to your $PATH environmental variable.
+### Download Autometa, setup compute environment and format marker databases
 
-If you want to calculate coverages, then also install bowtie2, samtools and bedtools:
 
-```
-sudo apt-get install bowtie2 bedtools
-wget https://github.com/samtools/samtools/releases/download/1.6/samtools-1.6.tar.bz2
-tar -vxjf samtools-1.6.tar.bz2
-cd samtools-1.6
-./configure --prefix=~/samtools
-make
-make install
+```bash
+# This will download Autometa into your current directory
+git clone https://github.com/KwanLab/Autometa.git
+# Navigate to Autometa directory
+cd Autometa
+# Create autometa environment
+conda create -n autometa -c bioconda -c conda-forge python=3.7 --file=requirements.txt
+## Activate your autometa environment
+conda activate autometa
+## Setup lca functions
+cd pipeline
+python setup_lca_functions.py build_ext --inplace
+## Setup HMM databases
+hmmpress -f single-copy_markers/Bacteria_single_copy.hmm 
```

Comment:
I'm not sure I understand this suggested change.

Currently the behavior is to navigate into the pipeline directory then to navigate to the previous directory after the lca_functions are setup. 

e.g.

```bash
cd pipeline # navigate to pipeline directory
... # do stuff
cd - # navigate to previous directory
... # continue on ...
```

Reply:
Ahh, I see. This is related to the README, not the Dockerfile. 👍

Reply:
The directory `single-copy_markers` is not in `pipeline`. Therefore for a user to follow along they need to do `cd ..` otherwise you will get e-mails about hmmpress having a file not found error.